CHANGELOG.md

```diff
@@ -5,6 +5,41 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+
+### Added
+
+- `cirrus-<stage>-workflow-event` SNS topic, and
+  `WorkflowEventManager` class for managing workflow event actions. ([#261])
+  The actions managed by this class include:
+
+  - updating state of workflows in `StateDB`
+  - announcing interactions cirrus has with a payload to the
+    `cirrus-<stage>-workflow-event`
+    Note: To use this topic, existing deployments will need to add the following
+    to their environment in both their `cirrus.yml` file:
+
+  ```yaml
+  CIRRUS_WORKFLOW_EVENT_TOPIC_ARN: !Ref WorkflowEventTopic
+  ```
+
+  and add the Topic to their `cloudformation/resources.yml` file:
+
+  ```yaml
+  # SNS Topic for failed workflows
```

Comment:
incorrect comment as it's not just failed workflows

Reply:
fixed in #263